addons/website/views/snippets.xml

```diff
@@ -353,6 +353,47 @@
     </section>
 </template>
 
+<template id="s_countdown" name="Countdown">
+    <section class="s_countdown"
+        data-display="dhms"
+        data-end-action="nothing"
+        data-layout="light"
+        data-size="175"
+        data-end-time="1604140819">
+        <div class="container">
+            <div class="row">
+                <div class="col text-center mt8 mb8">
+                    <span class="s_countdown_days">
+                        <canvas height="175" width="175"/>
+                    </span>
+                    <span class="s_countdown_hours">
+                        <canvas height="175" width="175"/>
+                    </span>
+                    <span class="s_countdown_minutes">
+                        <canvas height="175" width="175"/>
+                    </span>
+                    <span class="s_countdown_seconds">
+                        <canvas height="175" width="175"/>
+                    </span>
+                </div>
+            </div>
+            <div class="row s_countdown_end_message d-none">
+                <div class="col-12 text-center alert alert-info css_non_editable_mode_hidden o_not_editable" t-ignore="True" role="status">
+                    The following message will appear <strong>only</strong> once the countdown ends.
+                </div>
+                <div class="col-12 text-center">
+                    <p>Time's up! Visit our facebook page to know if you won a free ticket to our next summit.</p>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col d-none text-center s_countdown_end_redirect_message">
+                    Time's up! You can now visit <a class="s_countdown_end_redirect_url" href="">this page</a>.
+                </div>
+            </div>
```

Comment:
The redirect URL message is only shown when the user lands on a terminated countdown set to `Redirect`.
Indeed, it could have been a separate template only rendered when needed.
**Done**

About the message shown when `Show Message` is selected, this will remain as such in the DOM as discussed, as it is fully customizable and I think it is better to not lose the DOM on snippet option change.

Reply:
> About the message shown when Show Message is selected, this will remain as such in the DOM as discussed, as it is fully customizable and I think it is better to not lose the DOM on snippet option change.

You can implement the fact the message is not lost without saving unnecessary DOM :wink:

Reply:
Done, the countdown snippet now only contains the canvas elements.
The messages elements (both `redirect` message displayed once the countdown is over and the message shown in `show message` mode)